src/statement_generator.cpp

```diff
@@ -92,24 +92,43 @@ std::shared_ptr<GeneratorContext> StatementGenerator::GetDatabaseState(ClientCon
 	return result;
 }
 
+class RandomsConfig {
+public:
+	idx_t select_percentage = 60;
+	idx_t attach_percentage = 40;
+	idx_t attach_use_percentage = 50;
+	idx_t detach_percentage = 60;
+	idx_t set_percentage = 30;
+	idx_t delete_percentage = 40;
+
+	RandomsConfig(){
+
+	};
+
+	void getConfigFromFile() {
```

Comment:
You can also have a `GetDefaultConfig()` that gets some default configs using hardcoded values